Lacrit/assignment3/Dictionary.hpp

```diff
@@ -0,0 +1,42 @@
+#include <fstream>
+#include <iostream>
+#include <unordered_set>
+#include <string>
+
+
+class Dictionary {
+public:
+	Dictionary();
+    Dictionary(std::string);
+    ~Dictionary();
+	void Fill(std::string);
+	bool isPrefix(const std::string&); 
+
+    std::unordered_set<std::string> words;
+    std::unordered_set<std::string> prefixes;
+	std::string path;
+};
+
+Dictionary::Dictionary() {};
+Dictionary::Dictionary(std::string path):
+	path(path)
```

Comment:
It helps readability having different naming conventions for fields and parameters. On the Google Style for example, we end fields with '_'.